src/Grid/FormGrid/FormGridColumn.jsx

```diff
@@ -7,7 +7,7 @@ import { getDisplay } from '../renderer';
  *
  * @constructor
  */
-export const Column = (props) => {
+export const FormGridColumn = (props) => {
```

Comment:
ist erst einmal so ok. wir sollten aber am besten versuchen nur noch eine Column Komponente zu haben die wir in allen Grids einsetzen können

Reply:
ja, wir haben ja schon ein ticket dafür glaub ich um die Grid Funktionen zu vereinheitlichen